test/runtests.bat

```diff
@@ -10,7 +10,7 @@ REM compile the code into the bin folder
 javac  -cp ..\src -Xlint:none -d ..\bin ..\src\seedu\addressbook\Main.java
 
 REM run the program, feed commands from input.txt file and redirect the output to the actual.txt
-java -classpath ..\bin seedu.addressbook.Main < input.txt > actual.txt
+java -classpath ..\bin --add-modules java.xml.bind seedu.addressbook.Main < input.txt > actual.txt
```

Comment:
Is this necessary for the adding of the sort command? Not sure where the module is used.

Reply:
This is a quick fix for this missing module when running runtests.bat.